src/components/organisms/Header/index.tsx

```diff
@@ -0,0 +1,30 @@
+/** @jsx jsx */
+import { jsx, Text, Box, Flex } from 'theme-ui';
+
+import { IconHello } from 'assets/svg';
+import Inner from 'components/styles/Inner';
+import { HeaderProps } from './types';
+
+const Header: React.FC<HeaderProps> = ({ title, description }) => (
+  <Flex as="header" variant="header">
+    <Inner>
+      <Flex variant="header.content">
+        <Flex variant="header.icon">
+          <IconHello />
+        </Flex>
+        <Box>
+          <Text as="h1" variant="heading1" sx={{ variant: 'header.title' }}>
```

Comment:
You have a variant passed once already and it can be a bit misleading when you're using it again inside `sx`. Maybe in such cases, it would be better to use `styled` from `emotion` or add `heading1` to `header title variant`

Reply:
Done.